chainweb.cabal

```diff
@@ -425,7 +425,7 @@ library
 -- -------------------------------------------------------------------------- --
 
 test-suite chainweb-tests
-    import: warning-flags, debugging-flags
+    import: debugging-flags
```

Comment:
is this warning removal just for local dev? If so, I believe you can just add a cabal.project.local with the appropriate settings.

Reply:
Yep, that's what I do for local development, too lazy to explore `cabal.project.local`